packages/intent/src/skill-paths.ts

```
@@ -0,0 +1,77 @@
+import { existsSync } from 'node:fs'
+import { join, relative } from 'node:path'
+import type { SkillUse } from './skill-use.js'
+import type { SkillEntry } from './types.js'
+import { toPosixPath } from './utils.js'
```

ESLint

[error] `./utils.js` import should occur before type import of `./skill-use.js`

(import/order)